client/src/components/Allocations/Allocations.scss

```diff
@@ -13,118 +18,85 @@
-    font-family: Roboto, sans-serif;
 
-    /* !!!: This link is `40px`, which is taller than other sidebar links (which are ~`35px`) */
+    padding-top: var(--vertical-buffer);
+
     .nav-link {
         /* NOTE: These styles match `.nav-content`. That class is not used in
                  the markup for this sidebar because it is extraneous */
         /* RFE: Resolve discrepancy when common Sidebar component exists */
         display: flex;
         align-items: center;
 
+        min-width: var(--min-width);
+
         color: #707070;
-        padding-left: 1.5em; /* ~20px design * 1.2 design-to-app ratio (rounded) */
+        padding-left: var(--horizontal-buffer);
 
-        .link-icon {
-            font-size: 1.125rem; /* ~18px (16px design * 1.2 design-to-app ratio) (rounded) */
-            width: 1em;
-        }
         .link-text {
             font-size: 0.75rem; /* 12px (10px design * 1.2 design-to-app ratio) */
             font-weight: 500;
             padding-left: 16px;
-            vertical-align: middle;
         }
     }
     .active {
         background: rgba(#9d85ef, 0.25);
     }
 }
+.allocations-content {
+    /* As a flex child */
+    flex-grow: 1;
+
+    /* WARNING: Mimicked on: History, Allocation, DataFiles, DataFilesProjectsList, DataFilesProjectFileListing, PublicData */
+    padding-top: 1.75rem; /* ~28px (22.5px * design * 1.2 design-to-app ratio) */
+    padding-left: 1.5em; /* ~24px (20px * design * 1.2 design-to-app ratio) */
+}
 .allocations-table {
-    display: flex;
-    flex-direction: row;
-    width: 100%;
-    padding: 0 0 0 1em;
-    overflow-y: auto;
-    table {
-        width: 100%;
-        border: 0;
-        font-size: 14px;
+    /* Until Allocations uses <InfiniteScrollTable>, extend its styles */
+    /* TODO: Instead, update <AllocationsTable> to use <InfiniteScrollTable> */
+    @extend .InfiniteScrollTable;
+
+    tbody {
         tr {
-            height: 2.25rem;
-            td {
+            .system-cell {
                 padding: 0;
-                vertical-align: text-top;
             }
+
+            /* Override `.InfiniteScrollTable tbody tr…:not([class=-status])` */
+            background-color: transparent !important;
+
             border-bottom: 1px solid rgba(#707070, 0.25);
-            margin-bottom: 24px;
-        }
-        thead {
-            tr {
-                border-bottom: 1px solid #707070;
-            }
-        }
-        tbody {
-            tr:last-child {
-                border-bottom: none;
-            }
         }
         .sub-table-row {
+            width: 100%;
+
             /* FP-457: Will fix this *//* stylelint-disable no-descending-specificity */
             :nth-child(even) {
                 background-color: rgba(#c6c6c6, 0.1);
             }
             /* stylelint-enable no-descending-specificity */
             color: #707070;
-            font: 14px Roboto, sans-serif;
         }
```

Comment:
__Explanation__
Make `.allocations-table` mirror `.InfiniteScrollTable` until it is refactored to use it or a common Table component (that InfiniteScrollTable itself would also use).